pkg/analysis/symbol.go

```diff
@@ -0,0 +1,30 @@
+package analysis
+
+import (
+	"go.lsp.dev/protocol"
+
+	"github.com/tilt-dev/starlark-lsp/pkg/document"
+	"github.com/tilt-dev/starlark-lsp/pkg/query"
+)
+
+func (a Analyzer) SymbolAtPosition(doc document.Document, pos protocol.Position) query.Symbol {
+	var result query.Symbol
+	pt := query.PositionToPoint(pos)
+	nodes, ok := a.nodesAtPointForCompletion(doc, pt)
+	if !ok {
+		return result
+	}
+
+	limit := nodes[len(nodes)-1].EndPoint()
+	symbols := a.completeExpression(doc, nodes, limit)
```

Comment:
Still find it funny that we run the completion logic to find the symbol, but hey, it works.

Reply:
yeah, at some point we should probably rename it